docs/src/content/docs/tr/getting-started.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: Başlarken
+description: Sıradaki dokümantasyon site Astro'nun Starlight'ı ile nasıl oluşturacağını öğren.
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+## Yeni Proje Oluşturma
+
+Starlight, Astro çerçevesi üstüne kurulmuş çok özellikli bir dokümantasyon temasıdır.
+
+Aşağıdaki komutları kullanarak yeni bir Astro + Starlight projesi oluşturabilirsin:
+
+<Tabs>
+<TabItem label="npm">
+
+```sh
+# npm ile yeni bir proje oluştur
+npm create astro@latest -- --template starlight
+```
+
+</TabItem>
+<TabItem label="pnpm">
+
+```sh
+# pnpm ile yeni bir proje oluştur
+pnpm create astro --template starlight
+```
+
+</TabItem>
+<TabItem label="Yarn">
+
+```sh
+# yarn ile yeni bir proje oluştur
+yarn create astro --template starlight
+```
+
+</TabItem>
+</Tabs>
+
+Bu, siten için yapılandırmalar ve tüm gerekli dosyalarla birlikte yeni bir [proje dizini](/guides/project-structure/) oluşturacaktır.
```

Comment:
Links should be to the `/tr/` page so we don’t redirect people away from the Turkish version of the site:
```suggestion
Bu, siten için yapılandırmalar ve tüm gerekli dosyalarla birlikte yeni bir [proje dizini](/tr/guides/project-structure/) oluşturacaktır.
```